packages/devtools-evm/src/signer/sdk.ts

```diff
@@ -37,41 +64,91 @@ export class OmniSignerEVM implements OmniSigner {
         }
     }
 
-    #assertTransaction(transaction: OmniTransaction) {
-        assert(
-            transaction.point.eid === this.eid,
-            `Could not use signer for ${formatEid(this.eid)} to sign a transaction for ${formatOmniPoint(
-                transaction.point
-            )}`
-        )
-    }
-
     #serializeTransaction(transaction: OmniTransaction): TransactionRequest {
-        // Still missing
-        //
+        return {
+            // mandatory
+            to: transaction.point.address,
+            data: transaction.data,
 
-        // from?: string,
-        // nonce?: BigNumberish,
+            // optional
+            ...(transaction.gasLimit && { gasLimit: transaction.gasLimit }),
+            ...(transaction.value && { value: transaction.value }),
+        }
+    }
+}
 
-        // gasPrice?: BigNumberish,
+/**
+ * Implements an OmniSigner interface for EVM-compatible chains using Gnosis Safe.
+ */
+export class GnosisOmniSignerEVM<TSafeConfig extends ConnectSafeConfig> extends OmniSignerEVMBase {
+    // TODO: upgrade from @gnosis.pm to @safeglobal dependencies once the codebase upgrades to Ethers v6.  Currently,
+    // devtools only supports Ethers v5, and @safeglobal only supports Ethers v6.
 
-        // data?: BytesLike,
-        // chainId?: number
+    protected safeSdk: Safe | undefined
+    protected safeService: SafeServiceClient | undefined
 
-        // type?: number;
-        // accessList?: AccessListish;
+    constructor(
+        public override readonly eid: EndpointId,
+        public override readonly signer: Signer,
```

Comment:
You don't need the `public override readonly` since the parent class defines these, just pass them to `super`